.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: 2rd7sd9uaFAShnmJDTrL7TC9sChrDO2BJ
```

Comment:
This token shouldn't be exposed to the public. You can use environment variables in case you have something to do with it but I think for this case, you don't need it